matcher/matcher.go

```diff
@@ -131,17 +131,17 @@ func NewJoinSession(cfg *Config) *JoinSession {
 	return m
 }
 
-// Run listens for all matcher messages and runs the matching engine.
+// Run listens for all request messages from clients and runs the matching engine.
 func (matcher *JoinSession) Run() error {
 
 	for {
 
 		select {
-		// We use one timer to control whether participants send data in time
-		// With one progress, server has max time for client process is 2minutes (setting in config file)
+		// We use one timer to control participants sending data in time.
+		// With one progress, server has max time for client process is 30 seconds (setting in config file)
 		// After that time, client still not send data, server will consider client is malicious and remove
 		case <-matcher.clientTimeout.C:
-			// Check participants who missing sending data and remove
+			// Check participants who not sending data and remove.
```

Comment:
The updates to the comments made are better but still not there yet, grammar is still a problem. I've pointed out a couple of cases here. Take another look and correct.